CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+# dbt_jira v0.18.0
+[PR #131](https://github.com/fivetran/dbt_jira/pull/131) contains the following updates:
+## Breaking Changes
+> Since the following changes are breaking, a `--full-refresh` after upgrading will be required.
+
+- Changed the partitioning from days to weeks in the following models for BigQuery and Databricks All Purpose Cluster destinations:
+  - `int_jira__issue_calendar_spine`
```

Comment:
This is now no longer an incremental model. We should clarify that update in the breaking change release notes.

Reply:
Updated.